### PR TITLE
updated the document to add the volume provisioning documentation wit…

### PR DESCRIPTION
**Pull request by aadeshpa**
…h link to outside file

### CODE REVIEW

File: pipelines.adoc [hunk 1]
```diff
@@ -54,15 +54,9 @@ Explore how to use Kabanero Pipelines to build and manage Kabanero collections.
...
-
-Your persistent volume must be configured in accordance with your cloud provider’s default storage class. Otherwise, your pipelines might create a new volume for each run, which increases your pipeline run execution time.
-
-Alternatively, if you are using an unmanaged cluster, you can set up a persistent volume to be used by the pipeline. A simple persistent volume definition is provided in the following example. Update the path and other values in your `pv.yaml` file to suit your requirements. For instance, the following example is configured for local storage. You might edit this file to configure it for NFS, the standard configuration in a production environment.
+Tekton pipelines require a configured volume that is used by the framework to share data across tasks. The `build task`, which uses `Buildah`, also requires a volume mount. The pipelinerun should be creating a Persistent Volume Claim PVC with a requirement for 5 Gi persistent volume.
```
Comment (dmuelle) [inline pipelines.adoc:59]: **change**
The pipelinerun should be creating a Persistent Volume Claim PVC with a requirement for 5 Gi persistent volume.
**to**
The pipelinerun must create a Persistent Volume Claim (PVC) with a requirement of 5 Gi.

File: pipelines.adoc [hunk 2]
```diff
@@ -54,15 +54,9 @@ Explore how to use Kabanero Pipelines to build and manage Kabanero collections.
 
 === Getting started
 
-==== Set up dynamic volume provisioning or a persistent volume
+==== Setting up of a persistent volume for running pipelines
```
Comment (dmuelle) [inline pipelines.adoc:57]: **Change**
Setting up of a persistent volume for running pipelines
**to**
Set up  a persistent volume to run pipelines

File: pipelines.adoc [hunk 3]
```diff
@@ -73,13 +67,14 @@ Alternatively, if you are using an unmanaged cluster, you can set up a persisten
 . Clone the link:https://github.com/kabanero-io/kabanero-pipelines[kabanero-pipeplines repo]
 
   git clone https://github.com/kabanero-io/kabanero-pipelines
+  
+===== Static Persistent Volumes
 
-. Clone the `pv.yaml` file in this repo and apply it.
-
+If you are not running your openshift cluster on public cloud, you can setup the persistent volume statically. For example on the usage of static persistent volume provisioning you can go https://github.com/kabanero-io/kabanero-pipelines/blob/master/pipelines/docs/VolumeProvisioning.md#static-persistent-volumes[here]
```
Comment (dmuelle) [inline pipelines.adoc:73]: **change**
If you are not running your openshift cluster on public cloud, you can setup the persistent volume statically. For example on the usage of static persistent volume provisioning you can go https://github.com/kabanero-io/kabanero-pipelines/blob/master/pipelines/docs/VolumeProvisioning.md#static-persistent-volumes[here]

**to**
If you are not running your OpenShift cluster on a public cloud, you can set up a static persistent volume. For an example of how to use static persistent volume provisioning, see  https://github.com/kabanero-io/kabanero-pipelines/blob/master/pipelines/docs/VolumeProvisioning.md#static-persistent-volumes[Static Persistent Volumes].

File: pipelines.adoc [hunk 4]
```diff
@@ -73,13 +67,14 @@ Alternatively, if you are using an unmanaged cluster, you can set up a persisten
...
 
- cd ./pipelines/common
- oc apply -f pv.yaml -n kabanero
+===== Dynamic Volume Provisioning
 
+If you are running your openshift cluster on a public cloud, you can take advantage of dynamically provisioning the persistent volume.You can use your cloud provider’s default storage class for it. For example on the usage of dynamic persistent volume provisioning you can go https://github.com/kabanero-io/kabanero-pipelines/blob/master/pipelines/docs/VolumeProvisioning.md#dynamic-volume-provisioning[here]
```
Comment (dmuelle) [inline pipelines.adoc:77]: **change**
If you are running your openshift cluster on a public cloud, you can take advantage of dynamically provisioning the persistent volume.You can use your cloud provider’s default storage class for it. For example on the usage of dynamic persistent volume provisioning you can go https://github.com/kabanero-io/kabanero-pipelines/blob/master/pipelines/docs/VolumeProvisioning.md#dynamic-volume-provisioning[here].

**to**
If you run your OpenShift cluster on a public cloud, you can set up a dynamic persistent volume by using  your cloud provider’s default storage class. For an example of how to use dynamic persistent volume provisioning, see https://github.com/kabanero-io/kabanero-pipelines/blob/master/pipelines/docs/VolumeProvisioning.md#dynamic-volume-provisioning[Dynamic Volume Provisioning].

**Review by dmuelle — CHANGES_REQUESTED** [review]
Minor edits for IBM style

File: pipelines.adoc (hunk 2 above)
Comment (aadeshpa) [reply pipelines.adoc:57]: looks good.

File: pipelines.adoc (hunk 1 above)
Comment (aadeshpa) [reply pipelines.adoc:59]: looks good

File: pipelines.adoc (hunk 3 above)
Comment (aadeshpa) [reply pipelines.adoc:73]: looks good

File: pipelines.adoc (hunk 4 above)
Comment (aadeshpa) [reply pipelines.adoc:77]: looks good

**Review by aadeshpa — COMMENTED** [review]
looks good, you can go ahead and merge this.

**Review by dmuelle — APPROVED** [review]
File still needs editing for style and clarity- approving to merge for further tech update

**Review by aadeshpa — COMMENTED** [review]
looks good.

**Pull request merged**